### PR TITLE
Initial Enterprise docs

### PR DESCRIPTION
**Pull request by rkuchan**
Excuse the awful commit history. Rebase fail.

I use ascii art in the Users, Permissions, and Roles doc. Happy to remove it if everyone dislikes it.

### CODE REVIEW

File: content/enterprise/v1.0b/administration/users_permissions_roles.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ / \     ---> Permission F
+```
+
+## Permissions
+
+A permission is a grant for access.
+They can be global, cluster specific, or database specific.
+Permissions can be granted to roles and to users.
+
+TODO: Confirm
```
Comment (beckettsean) [inline content/enterprise/v1.0b/administration/users_permissions_roles.md]: Yes, let's confirm with @markbates that these are correct.

File: content/enterprise/v1.0b/introduction/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  enterprise_1b:
+    weight: 10
+    parent: introduction
+---
+
+Now that you've successfully
+[installed](/enterprise/v1.0b/introduction/installation/) InfluxEnterprise,
+it's time to start working with the app.
+
+This document walk you through:
```
Comment (beckettsean) [inline content/enterprise/v1.0b/introduction/getting_started.md]: Link to Zendesk private help center, to a particular article if possible.

File: content/enterprise/v1.0b/introduction/installation.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    weight: 0
+    parent: introduction
+---
+
+The Enterprise web application is TODO: cool description here.
+This document gets you up and running with the Enterprise web application.
+
+### Requirements
+
+* Your InfluxEnterprise [license key](TODO: LINK HERE)
```
Comment (beckettsean) [inline content/enterprise/v1.0b/introduction/installation.md]: link goes to Enterprise Beta sign-up form.

**Comment by markbates** [comment]
@timraymond since you coded this, can you confirm the user permissions stuff here is correct?

File: content/enterprise/v1.0b/administration/users_permissions_roles.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+| Drop Databases | The right to drop [databases](/influxdb/v1.0/concepts/glossary/#database) and [retention policies](/influxdb/v1.0/concepts/glossary/#retention-policy-rp). | X | X | |
+| Manage Continuous Queries | The right to create, list, and drop [continuous queries](/influxdb/v1.0/concepts/glossary/#continuous-query-cq). | X | X | X |
+| Manage Queries | The right to `SHOW` and `KILL` queries. | | | |
+| Manage Shards | The right to `SHOW` and `DROP` shards. | | | |
+| Manage Subscriptions | The right to `SHOW`, `ADD`, and `DROP` subscriptions. | | | |
+| Monitor | The right to `SHOW STATS` and `SHOW DIAGNOSTICS`. | | | |
+| Read Data | The right to read data. | X | X | X |
+| View Admin | The right to view the admin screen and diagnostics dashboard. | NA | NA | NA |
+| View Chronograf | The right to view Chronograf screens. | NA | NA | NA |
+| Write Data | The right to write data. | X | X | X |
```
Comment (timraymond) [inline content/enterprise/v1.0b/administration/users_permissions_roles.md]: There are two additional permissions that were added recently and haven't made it into the UI. Those are "Copy Shard" and "Rebalance." The former will allow users to issue copy shard commands via `influx-ctl`, and the latter will allow them to use the "Rebalance" button to rebalance the cluster.

**Pull request closed without merging**